modules/db/src/main/scala/gem/dao/UserDao.scala

```diff
@@ -81,11 +110,39 @@ object UserDao {
       """.query
 
     def changePassword(uid: User.Id, oldPassword: String, newPassword: String): Update0 =
-      sql"""
+      (fr"""
         UPDATE gem_user
-        SET    md5 = md5($newPassword)
-        WHERE  id  = ${uid}
-        AND    md5 = md5($oldPassword)
+        SET    hash = crypt($newPassword, gen_salt('bf', 10))
```

Comment:
Nice, I didn't realize `pgcrypto` already included the `bcrypt` algorithm. I wonder why they don't highlight it in the documentation.